### PR TITLE
Feature/wcag componentpreview fieldset

### PR DESCRIPTION
**Pull request by eskilsen**
## Description

From FUNKA audit we found out the componentpreview didn't use html semantics. Options menu is now updated to legend and fieldset instead of h4 and divs. 

Task: https://payexjira.atlassian.net/jira/software/c/projects/SWED/boards/312?modal=detail&selectedIssue=SWED-1051

## How Has This Been Tested?

Ran all existing tests and update snapshots. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/App/routes/patterns.js [hunk 1]
```diff
@@ -29,6 +29,14 @@ module.exports = [
...
                 outlined: true,
                 statusBadges: ["new"]
+            },
+            {
+                title: "Filters",
+                path: "/patterns/filters",
+                componentPath: "content/Filters",
+                icon: "filter_alt",
+                outlined: true,
+                statusBadges: ["new"]
```
Comment (PhungNg) [inline src/App/routes/patterns.js:39]: 😅

**Review by PhungNg — COMMENTED** [review]
Very nice! 🏅 
Just a small change on the Dropdown section. And the filter part

**Review by PhungNg — CHANGES_REQUESTED** [review]
Forgot to request a change 😅

File: src/App/routes/patterns.js (hunk 1 above)
Comment (eskilsen) [reply src/App/routes/patterns.js:39]: 😠

**Review by PhungNg — APPROVED** [review]
Noice👍

**Pull request merged**